src/d3a/models/strategy/pv.py

```diff
@@ -68,22 +67,21 @@ def __init__(self, panel_count: int = 1,
         self.panel_count = panel_count
         self.max_panel_power_W = max_panel_power_W
         self.state = PVState()
+        self.use_market_maker_rate = use_market_maker_rate
+        self.final_selling_rate = final_selling_rate
```

Comment:
The external pv strategy is inheriting from `PVStrategy` where we dont need these variables. This is why they are initiated in a separate function. Not really important, but IMO, please revert.

Reply:
That's interesting...I noticed this inconsistency because pylint was complaining about it. I would normally expect a class to declare all instance variables in `__init__()`, also because the `__init__()` usually tells which instance variables we can expect and use.

This might be a sign that the current inheritance structure could be somehow improved (maybe with a new class). Anyway, I can revert this change.